content/docs/reference/images.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Dockerfile Locations"
+description = "Where to find the Dockerfiles for all of Kubeflow's images"
+weight = 10
++++
+
+| Image Name        | Dockerfile Location |
+| ------------- |---------------|
+| tf_operator      | <<https://github.com/kubeflow/tf-operator/tree/master/build/images/tf_operato>r> |
```

Comment:
There are a few too many angle-brackets on this line and the next, which are causing rendering problems in the links. See preview:
https://deploy-preview-917--competent-brattain-de2d6d.netlify.com/docs/reference/images/